ansible/roles/email-server-config/tasks/main.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Install required packages
+  package:
+    name: '{{ item }}'
+    state: present
+  with_items: '{{ required_packages }}'
+
+- name: Remove any exisitng postfix configs
+  file:
+    path: "/etc/postfix/{{ item }}"
+    state: absent
+  with_items: '{{ postfix_config }}'
+
+- name: Install postfix config configs
+  copy:
+    src: "{{ item }}"
+    dest: "/etc/postfix/{{ item }}"
+    mode: 0644
+  with_items: '{{ postfix_config }}' 
```

Comment:
Are we configuring anything in the locally stored config files?
If they are custom then it might be worth seeing whats different and using lineinfile/replace to add the config in rather than storing the whole files.

Reply:
Disclaimer and disclaimer.txt are new files and will need to be copied in. I have removed the other files and used inline file commands.